udata_front/theme/gouvfr/assets/js/pages/PublishingForm/Step2DescribeDataset.vue

```diff
@@ -441,6 +458,10 @@ function setGranularity(value: string) {
 function submit() {
   validateRequiredRules().then(valid => {
     if(valid) {
+      if (toValue(userOrganization) !== "user") {
```

Comment:
I think the case  `user` should be treated here too. Otherwise if you choose an organization and move to the next step, you can never go back to the user publishing.